web/src/components/molecules/Common/Form/AssetItem/index.tsx

```diff
@@ -69,14 +72,31 @@ const AssetItem: React.FC<Props> = ({
     fileList,
   };
 
+  useEffect(() => {
+    setAssetValue(assetList.find(asset => asset.id === defaultValue));
+  }, [defaultValue, assetList, setAssetValue]);
```

Comment:
Its better to just pass the asset as the default value, rather than passing the full assetList and a defaultValue and then assigning it here. It over complicates this component.

Reply:
@KaWaite , `assetList` is passed by default for the modal (By Nour when he implemented the modal).

Reply:
Ah sorry, didn't realize the assetList is also passed to the Modal. I thought it could be removed completely.